app/views/forever_style_guide/sections/components/navigation/_nav_account_dropdown_menu.erb

```diff
@@ -81,7 +81,7 @@
 
         <li class="dropdown-menu-account-item">
 
-          <%= link_to current_admin ? stop_impersonating_url : log_out_url, class: 'dropdown-menu-nav-link dropdown-menu-nav-link-sm dropdown-menu-nav-link-no_margin', title: 'Log Out of Forever' do %>
+          <%= link_to current_admin ? stop_impersonating_url : log_out_url, class: 'dropdown-menu-nav-link dropdown-menu-nav-link-sm dropdown-menu-nav-link-no_margin', title: 'Log Out of Forever', method: current_admin ? :delete : :post do %>
```

Comment:
@schweitzdesign not 100% sure this won'r produce errors since I don't have the store working locally :/ but if its not erroring not sure this is the right way to handle the conditional delete method?